packages/env/src/lib/env.ts

```diff
@@ -47,6 +47,17 @@ const getAppEnvVar = (name: string): string => {
       return typeof process !== "undefined"
         ? process.env.NEXT_PUBLIC_LAUNCHDARKLY_CLIENT_ID_PROD
         : import.meta.env.VITE_LAUNCHDARKLY_CLIENT_ID_PROD;
+    case "REFERRALHERO_ARTIST_REFERRAL_CAMPAIGN_UUID_STAGING":
+      return typeof process !== "undefined"
+        ? process.env
+            .NEXT_PUBLIC_REFERRALHERO_ARTIST_REFERRAL_CAMPAIGN_UUID_STAGING
+        : import.meta.env
+            .VITE_REFERRALHERO_ARTIST_REFERRAL_CAMPAIGN_UUID_STAGING;
+    case "REFERRALHERO_ARTIST_REFERRAL_CAMPAIGN_UUID_PROD":
+      return typeof process !== "undefined"
+        ? process.env
+            .NEXT_PUBLIC_REFERRALHERO_ARTIST_REFERRAL_CAMPAIGN_UUID_PROD
+        : import.meta.env.VITE_REFERRALHERO_ARTIST_REFERRAL_CAMPAIGN_UUID_PROD;
```

Comment:
@scandycuz  I'm just following the pattern here, but is this specific pattern necessary since this won't have a next/marketplace integration? Is there a different pattern that should be followed?